### PR TITLE
en,zh: add release notes for v1.3.2

### PR DESCRIPTION
**Pull request by just1900**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/releases/release-1.3.2.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+# TiDB Operator 1.3.2 Release Notes
+
+Release date: March 17, 2022
+
+TiDB Operator version: 1.3.2
+
+## Improvements
+
+- Support for TiDB to run on Istio enabled kubernetes cluster ([#4445](https://github.com/pingcap/tidb-operator/pull/4445), [@rahilsh](https://github.com/rahilsh))
```
Comment (ran-huang) [inline en/releases/release-1.3.2.md:13]:
```suggestion
- Support for TiDB to run on Istio-enabled kubernetes clusters ([#4445](https://github.com/pingcap/tidb-operator/pull/4445), [@rahilsh](https://github.com/rahilsh))
```

File: en/releases/release-1.3.2.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- Support Multi-arch docker image ([#4469](https://github.com/pingcap/tidb-operator/pull/4469), [@better0332](https://github.com/better0332))
```
Comment (ran-huang) [inline en/releases/release-1.3.2.md:14]:
```suggestion
- Support multi-arch docker image ([#4469](https://github.com/pingcap/tidb-operator/pull/4469), [@better0332](https://github.com/better0332))
```

**Review by ran-huang — COMMENTED** [review]
LGTM

File: zh/releases/release-1.3.2.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+TiDB Operator 版本：1.3.2
+
+## 优化提升
+
+- 支持在启用了 Istio 的 Kubernetes 集群上部署与运行 TiDB ([#4445](https://github.com/pingcap/tidb-operator/pull/4445), [@rahilsh](https://github.com/rahilsh))
+- 支持包括 ARM 系统架构在内的多架构 Docker 镜像 ([#4469](https://github.com/pingcap/tidb-operator/pull/4469), [@better0332](https://github.com/better0332))
+
+## Bug 修复
+
+- 修复 Discovery 服务错误导致 TiDB 集群 PD 组件启动失败的问题 ([#4440](https://github.com/pingcap/tidb-operator/pull/4440), [@liubog2008](https://github.com/liubog2008))
```
Comment (csuzhangxc) [inline zh/releases/release-1.3.2.md:18]: 这个在 1.3.1 修复的

Comment (just1900) [reply zh/releases/release-1.3.2.md:18]: 我印象中也是，我 revert 一下，1.3.1 release notes没找到😂

**Review by csuzhangxc — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: en/releases/release-1.3.2.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+title: TiDB Operator 1.3.2 Release Notes
+---
+
+# TiDB Operator 1.3.2 Release Notes
+
+Release date: March 17, 2022
```
Comment (ran-huang) [inline en/releases/release-1.3.2.md:7]:
```suggestion
Release date: March 18, 2022
```

File: zh/releases/release-1.3.2.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+title: TiDB Operator 1.3.2 Release Notes
+---
+
+# TiDB Operator 1.3.2 Release Notes
+
+发布日期: 2022 年 3 月 17 日
```
Comment (ran-huang) [inline zh/releases/release-1.3.2.md:7]:
```suggestion
发布日期: 2022 年 3 月 18 日
```

**Pull request merged**